examples/application_epg/main.tf

```diff
@@ -13,35 +13,149 @@ provider "aci" {
   insecure = true
 }
 
-resource "aci_tenant" "dev_tenant" {
-  name = "demo_tenant"
+resource "aci_tenant" "terraform_tenant" {
+  name        = "static_leaf_tenant"
+  description = "This tenant is created by terraform"
 }
 
 resource "aci_application_profile" "test_ap" {
-  tenant_dn   = aci_tenant.dev_tenant.id
-  name        = "demo_ap"
-  annotation  = "tag"
+  tenant_dn   = aci_tenant.terraform_tenant.id
+  name        = "test"
   description = "from terraform"
   name_alias  = "test_ap"
   prio        = "level1"
 }
 
-resource "aci_application_epg" "fooapplication_epg" {
+resource "aci_bridge_domain" "terraform_bd" {
+  tenant_dn = aci_tenant.terraform_tenant.id
+  name      = "test_bd"
+}
+
+resource "aci_rest" "rest_qos_custom_pol" {
+  path       = "api/node/mo/${aci_tenant.terraform_tenant.id}/qoscustom-testpol.json"
+  class_name = "qosCustomPol"
+
+  content = {
+    "name" = "testpol"
```

Comment:
suggestion:
`"name" = "test_pol"`

Apply it to the rest of the objects. For consistency again.